

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m11:09:34  Running with dbt=1.7.18
[0m11:09:34  Registered adapter: firebolt=1.6.4
[0m11:09:34  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m11:09:34  
[0m11:09:35  Concurrency: 1 threads (target='default')
[0m11:09:35  
[0m11:09:35  1 of 1 START seed file seed .................................................... [RUN]
[0m11:09:36  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 0.85s]
[0m11:09:36  
[0m11:09:36  Finished running 1 seed in 0 hours 0 minutes and 1.49 seconds (1.49s).
[0m11:09:36  
[0m11:09:36  [32mCompleted successfully[0m
[0m11:09:36  
[0m11:09:36  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m11:09:36  Running with dbt=1.7.18
[0m11:09:36  Registered adapter: firebolt=1.6.4
[0m11:09:36  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m11:09:36  
[0m11:09:36  Concurrency: 1 threads (target='default')
[0m11:09:36  
[0m11:09:36  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m11:09:37  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 0.81s]
[0m11:09:37  
[0m11:09:37  Finished running 1 incremental model in 0 hours 0 minutes and 1.46 seconds (1.46s).
[0m11:09:37  
[0m11:09:37  [32mCompleted successfully[0m
[0m11:09:37  
[0m11:09:37  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m11:09:37  Running with dbt=1.7.18
[0m11:09:37  Registered adapter: firebolt=1.6.4
[0m11:09:37  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m11:09:37  
[0m11:09:38  Concurrency: 1 threads (target='default')
[0m11:09:38  
[0m11:09:38  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m11:09:39  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 0.79s]
[0m11:09:39  
[0m11:09:39  Finished running 1 incremental model in 0 hours 0 minutes and 1.44 seconds (1.44s).
[0m11:09:39  
[0m11:09:39  [31mCompleted with 1 error and 0 warnings:[0m
[0m11:09:39  
[0m11:09:39    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "1e0b3d2b-bf34-4162-af25-3fae2a8405a0",
      "request_id": "f6188680-d4e1-4df3-8fb9-0291a0d4bb23"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m11:09:39  
[0m11:09:39  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
